crates/spartan-farmer/src/commands/dht/client.rs

```diff
@@ -0,0 +1,308 @@
+use super::*;
```

Comment:
Avoid `super::*` in all cases. One notable exception to this rule is tests, there you can use `use super::*` if tests are in the same file, otherwise prefer importing elements directly. Also don't import from other modules stuff that come from external dependencies, import it from there instead.